x-pack/plugins/transform/public/app/sections/create_transform/components/step_details/step_details_form.tsx

```diff
@@ -339,6 +424,28 @@ export const StepDetailsForm: FC<Props> = React.memo(
               data-test-subj="transformCreateIndexPatternSwitch"
             />
           </EuiFormRow>
+          {createIndexPattern && !indexPatternTitleExists && previewDateColumns.length > 0 && (
+            <EuiFormRow
+              helpText={i18n.translate(
```

Comment:
I think it would be useful to add a `label` for this form control, as they have for the Create Index Pattern page:

![image](https://user-images.githubusercontent.com/7405507/84398560-03ab0480-abf8-11ea-9120-ec47e4368bec.png)

Reply:
Added in https://github.com/elastic/kibana/pull/68842/commits/3fe458e73e36261939a364c1a1101626b85ba7b1

![Screen Shot 2020-06-11 at 10 24 43 AM](https://user-images.githubusercontent.com/43350163/84405635-098bf000-abce-11ea-8bae-71f2940d0cf0.png)